x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/EsqlActionTaskIT.java

```diff
@@ -507,6 +507,7 @@ protected void doRun() throws Exception {
     }
 
     public void testTaskContentsForTopNQuery() throws Exception {
+        nodeLevelReduction = false;
```

Comment:
I *think* we'd be better off forcing this to `true` rather than `false` for this test. We'd have to change it, but I think it'd be more descriptive of reallity.